coderd/database/migrations/000249_email_reports.down.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM notification_templates WHERE id = 'b02ddd82-4733-4d02-a2d7-c36f3598997d';
```

Comment:
This is a different UUID than in the up migration?

Reply:
Yes, good catch! I removed this migration now, but will address it in the next PR. Thank you.